tooling/lsp/src/requests/hover.rs

```diff
@@ -705,7 +705,7 @@ mod hover_tests {
             "workspace",
             "two/src/lib.nr",
             Position { line: 51, character: 8 },
-            &format!("    let x: BoundedVec<SubOneStruct, 3>\n\nGo to [SubOneStruct](file://{}#L4,12-4,24)", workspace_on_src_lib_path),
+            &format!("    let x: BoundedVec<SubOneStruct, (3: numeric u32)>\n\nGo to [SubOneStruct](file://{}#L4,12-4,24)", workspace_on_src_lib_path),
```

Comment:
```suggestion
            &format!("    let x: BoundedVec<SubOneStruct, 3>\n\nGo to [SubOneStruct](file://{}#L4,12-4,24)", workspace_on_src_lib_path),
```
To go along with the previous Display suggestion